### ISSUE TITLE
Japanese (Mozc) input method missing

### ISSUE THREAD

**Issue by jacobgkau** [pop-os/mozc#1]
This package was forked and modified because it was setting Japanese as a default input method, which we don't want. However, this had the side effect of removing the `Japanese (Mozc)` option from fresh installs:

![Screenshot from 2021-06-30 08-54-18](https://user-images.githubusercontent.com/7199422/123983071-cfc88b80-d980-11eb-976a-e6303932c43c.png)

After reverting to Ubuntu's version of `ibus-mozc` and `mozc-server`, the option is back, but it's activated by default (English is still the default input method, but the input method switcher is displayed and Japanese is an option without having to add it through Settings):

![Screenshot from 2021-06-30 08-52-08](https://user-images.githubusercontent.com/7199422/123982779-8d9f4a00-d980-11eb-9441-520865ae84cd.png)

On a machine that was upgraded from 20.10, with our version of the `mozc` packages now installed, I still have the option in Settings, but it's not added by default (no input method switcher is shown by default):

![Screenshot from 2021-06-30 08-55-00](https://user-images.githubusercontent.com/7199422/123983200-ea026980-d980-11eb-89cc-483027e077e2.png)

So it seems like the setup script we removed in our fork might be required to get the input method to show up in Settings. There should ideally be a way to make it show up in Settings without activating it by default.

**Comment by ids1024** [comment]
> On a machine that was upgraded from 20.10, with our version of the mozc packages now installed, I still have the option in Settings, but it's not added by default (no input method switcher is shown by default):

Hm, so the behavior is correct on upgrade, but not for a fresh install? Interesting.

**Comment by jacobgkau** [comment]
> Hm, so the behavior is correct on upgrade, but not for a fresh install? Interesting.

That's correct. Also, on that machine where I installed Ubuntu's version of the two mozc packages, after upgrading back to our versions, I still have the option in Settings. So it definitely seems like something to do with setup/initialization.

**Comment by Wazhai** [comment]
Fixing this issue really should be prioritized because Pop is still missing a user-friendly way to set up a Japanese IME like in other Gnome distros.

**Comment by jcayzac** [comment]
New Pop user here. I was confused as to why I couldn't setup Japanese input. What is the proper workaround for now?

**Comment by sunjay140** [comment]
I'm also having this issue with openSUSE & Gnome 44. The options to switch input method appears at launch but disappears after some time.
![Screenshot from 2023-07-10 22-41-03](https://github.com/pop-os/mozc/assets/38516947/c35536a4-e0a3-42df-bf20-6485808ba28e)

**Comment by jacobgkau** [comment]
> I'm also having this issue with openSUSE & Gnome 44.

@sunjay140 This repository is for the mozc fork being used in Pop!_OS. I don't think it would be relevant to an issue on openSUSE, unless you're suggesting the issue is somewhere upstream.
